actioncable/actioncable.gemspec

```diff
@@ -24,9 +24,12 @@ Gem::Specification.new do |s|
   s.add_dependency 'nio4r',            '~> 1.2'
   s.add_dependency 'websocket-driver', '~> 0.6.1'
 
-  s.add_development_dependency 'em-hiredis', '~> 0.3.0'
+  s.add_development_dependency 'coffee-script',        '~> 2.4.1'
+  s.add_development_dependency 'coffee-script-source', '~> 1.10.0'
```

Comment:
Sorry, I didn't realize which gem you were asking about. The `coffee-script` gem [depends on `coffee-script-source`](https://github.com/josh/ruby-coffee-script/blob/master/coffee-script.gemspec#L19), but doesn't declare a version. Specifying one here ensures the `.coffee` files are compiled consistently.